apps/test-bundles/package.json

```diff
@@ -1,45 +1,25 @@
 {
   "name": "test-bundles",
   "version": "7.0.0",
-  "description": "Testing the button for bundling.",
-  "main": "lib-commonjs/index.js",
-  "module": "lib/index.js",
-  "sideEffects": [
-    "lib/version.js"
-  ],
-  "typings": "lib/index.d.ts",
+  "description": "Testing Fluent UI React bundle sizes.",
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "just-scripts build",
```

Comment:
we should keep `build`?

Reply:
I'm pretty sure the only things to build in this package were the TS files for the old bundle size thing, which I didn't think were being used at all. Though now I see size auditor is consistently failing with weird lightrail errors, so maybe I was wrong about that?

Reply:
Hmm the lightrail job is failing with similar errors on one of your PRs, so maybe it's unrelated https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=135115&view=results